pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeGraphs/dashboards/crossClusterReplication.tsx

```diff
@@ -48,5 +49,32 @@ export default function (props: GraphDashboardProps) {
         />
       </Axis>
     </LineGraph>,
+    <LineGraph
+      title="Replication Lag"
+      sources={storeSources}
+      tenantSource={tenantSource}
+      tooltip={`Replication lag between primary and standby cluster`}
+    >
+      <Axis units={AxisUnits.Duration} label="duration">
+        <Metric
+          downsampler={TimeSeriesQueryAggregator.MIN}
+          aggregator={TimeSeriesQueryAggregator.AVG}
+          name="cr.node.physical_replication.replicated_time_seconds"
+          title="Replication Lag"
+          transform={datapoints =>
+            datapoints.map(d =>
```

Comment:
Do we need to wrap this in a `map`? `transform` seems to be equivalent to `map` so we can just pass the inner function here.